cmake/testing.cmake

```diff
@@ -126,10 +126,15 @@ function (rp_test)
       set(gtest_option "--gtest")
   endif()
 
+  set(root_option "--root /dev/shm/vectorized_io")
```

Comment:
I don't think any rely on it, but they all use it. I'm pretty sure it was added because some tests that were doing a lot of small fsync writes would run very slowly on EBS drives or something. Were you thinking it'd be good just to have everything use real disks?

Reply:
Yeah maybe? Its fine to leave for now.